ruby/vampires.rb

```diff
@@ -0,0 +1,99 @@
+#Ask for number of repetitions
+puts "How many individuals need to take the HR Interview Survey today?"
+survey_reps = gets.chomp.to_i
+
+until survey_reps == 0 
+  # Ask new employees their name,
+  puts "Welcome to the Werewolf Inc. Interview Survey!"
+  puts "First, what is your full legal name?"
+  employee_name = gets.chomp.upcase
+  
+  # Age & birth year
+  puts "Next, what is your age in years?"
+  employee_age = gets.chomp.to_i
+  
+  puts "In what year were you born?"
+  birth_year = gets.chomp.to_i
+  
+  real_age = 2017 - birth_year
+  
+  # If they want garlic bread from cafeteria
+  
+  puts "[Y/N] I am interested in ordering garlic bread from the company cafeteria"
+  garlic_preference = gets.chomp.upcase
+  
+  # If they want to enroll in company health insurance
+  puts "[Y/N] Do you plan on enrolling in our company health insurance plan?"
+  health_plan = gets.chomp.upcase
+  
+  allergy_input = nil
+  until allergy_input == "done" || allergy_input == "sunshine"
+    puts "Please list all allergies one at a time, and enter 'done' when you are finished"
+    
+    allergy_input = gets.chomp.downcase
+    
+    if allergy_input == "sunshine"
+    puts "Probably a vampire!"
+    end
+    
+  end 
+  
+  
+  puts ""
+  puts "SURVEY RESULTS:"
+  
+  # #Detection Logic
+  unlikely_vampire = false
```

Comment:
Instead of creating 4 separate variables to represent each state, why not use 1 variable and assign the string you are going to return as the checks take place?